README.md

```diff
@@ -32,12 +32,16 @@ Q2: What is the answer?
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+*_Jeremy_: I originally started my career at Nationwide as a requirements analyst and when I was hired on as an associate, I was hired as a developer. I enrolled in this program as a means to bridge any knowledge gap I held in regards to my new position.
 
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+*_Jeremy_: I like to travel as well with friends and family. Out of all the places I have been to so far, I liked Bergen, Norway the most due to the amazing scenes of nature. Next, I plan on going to Jamaica in the spring of 2020.
```

Comment:
Please add a little more detail about Bergen.  I don't really know anything about it.  Is it along the coast or inland?  What kind of nature... woodlands, glaciers, etc.?

Reply:
Just updated the pull request with some more details about Bergen.